scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -182,6 +185,21 @@
                      'containers using docker even when the runtime is detected to be something different.',
                      convert_to=bool, default=False, env_aware=True)
 
+define_config_option( __monitor__, 'k8s_use_controlled_warmer',
```

Comment:
Should these all be moved to the global config level, because presumably the controlled warmer is going to be used eventually for the kubernetes events monitor too?  Otherwise we'd need to duplicate the settings.

Another benefit of moving it to the global config is that it makes it easier to configure the cache to initialize straight away if it detects `k8s_use_controlled_warmer` is `True`